content/md/en/docs/tutorials/light-clients/parachains.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Connect with a parachain
```

Comment:
gatsby  mangles things with the same file name - can this file be renamed to to something less generic than `parachains.md`?

Reply:
I initially tried `parachain.md` and indeed there was some crazy behavior.
Maybe `connect-with-a-parachain.md`, together with `connect-with-a-relay-chain.md`?